packages/tabs/src/Tabs.ts

```diff
@@ -116,6 +113,10 @@ export class Tabs extends Focusable {
         this.setAttribute('role', 'tablist');
         this.addEventListener('mousedown', this.manageFocusinType);
         this.addEventListener('focusin', this.startListeningToKeyboard);
+        const selectedChild = this.querySelector('[selected]') as Tab;
```

Comment:
wondering if maybe selectedChild could be a `@query` property?

```
@query('[selected]')
private selectedChild?: Tab;
```

Reply:
_If_ this were possible in this context, I'd very much want to use it as well! However, the `@query` decorator only queries the element's render root, which would be `this.shadowRoot` in the context of our elements, and would not query for the elements in question here. There is an issue at the `LitElement` level to expand on the `@queryAsssignedNodes` decorator in a way that should support this technique: https://github.com/Polymer/lit-element/issues/1016

Reply:
Thinking this through a little there is some helper code that we _might_ use here:
```
        this.addressSelectedChild((child: Tab): void  => { this.selectTarget(child); });
    }

    private addressSelectedChild(addressWith: (child: Tab) => void): void {
        const selectedChild = this.querySelector('[selected]') as Tab;
        if (selectedChild) {
            addressWith(selectedChild);
        }
    }
```
I'm a little worried that this becomes harder to read, however. Let me know what you think!